CharlieBackend.Business/Helpers/ResponseMessages.cs

```diff
@@ -1,12 +1,15 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace CharlieBackend.Business.Helpers
+﻿namespace CharlieBackend.Business.Helpers
 {
     public class ResponseMessages
```

Comment:
make this class static, we dont need instance this class

Reply:
Done